libs/coin-modules/coin-bitcoin/src/exchange.ts

```diff
@@ -9,26 +9,21 @@ const addressFormatMap = {
   bitcoin_cash: 3,
   bech32m: 4,
 };
+type FormatType = keyof typeof addressFormatMap;
 
-const getSerializedAddressParameters = (
-  path: string,
-  derivationMode: DerivationMode,
+export const getSerializedAddressParameters = (
+  account: Account,
   addressFormat?: string,
-): {
-  addressParameters: Buffer;
-} => {
+): Buffer => {
+  const path = account.freshAddressPath;
+  const derivationMode = account.derivationMode;
+
   const format =
     addressFormat && addressFormat in addressFormatMap
-      ? addressFormat
-      : getAddressFormatDerivationMode(derivationMode);
+      ? (addressFormat as FormatType)
+      : (getAddressFormatDerivationMode(derivationMode) as FormatType);
```

Comment:
We should be able to avoid the `as` keyword here if we use an helper for the includes